mobile/src/components/common/date-picker/date-picker.tsx

```diff
@@ -57,7 +57,7 @@ const DatePicker = <T extends FormControlValues>({
 
   return (
     <View>
-      <Text style={styles.label}>{label}</Text>
+      {label && <Text style={styles.label}>{label}</Text>}
```

Comment:
always convert string values to boolean in conditions, especially in React Native, if label is empty string it will crash the app
```suggestion
      {hasLabel && <Text style={styles.label}>{label}</Text>}
```

Reply:
Can i do it like this `{label ? <Text style={styles.label}>{label}</Text> : null}` ?

Reply:
Or `{Boolean(label) && <Text style={styles.label}>{label ?? ''}</Text>}`

Reply:
it's better to move Boolean(label) outside

Reply:
<img src="https://gyazo.com/085c9b7a2a9b644500cd8d8de1077cb4.png" width="45%">